pontoon/sync/changeset.py

```diff
@@ -313,12 +326,20 @@ def execute_update_db(self):
         if self.changes['update_db']:
             entities_with_translations = self.prefetch_entity_translations()
 
+        updated_entities = []
         for locale_code, db_entity, vcs_entity in self.changes['update_db']:
             for field, value in self.get_entity_updates(vcs_entity, db_entity).items():
                 setattr(db_entity, field, value)
 
             if db_entity.is_dirty(check_relationship=True):
                 self.entities_to_update.append(db_entity)
+                updated_entities.append((
+                    db_entity.pk,
+                    db_entity.string,
+                    ' '.join(get_singulars(db_entity.string)),
+                    db_entity.string_plural,
+                    ' '.join(get_singulars(db_entity.string_plural))
```

Comment:
Commas at the end of all lines make diffs easier to read, next time someone adds a new line here. :)